pkgs/development/python-modules/pymatgen/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildPythonPackage, fetchPypi, glibcLocales, numpy, pydispatcher, sympy, requests, monty, ruamel_yaml, six, scipy, tabulate, enum34, matplotlib, palettable, spglib, pandas }:
+
+buildPythonPackage rec {
+  pname = "pymatgen";
+  version = "2018.6.27";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "8078af7fda4f9a07f1e389ffe08de3511213acdf9fb2ed9f9ffe89b9b12b8568";
+  };
+
+  nativeBuildInputs = [ glibcLocales ];
+  propagatedBuildInputs = [ numpy pydispatcher sympy requests monty ruamel_yaml six scipy tabulate enum34 matplotlib palettable spglib pandas ];
+  
+  # No tests in pypi tarball.
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    description = "A robust materials analysis code that defines core object representations for structures and molecules with support for many electronic structure codes.";
```

Comment:
https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes
I also think this is too long.